community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/StringSchemaKey.java

```diff
@@ -38,11 +38,16 @@
 
     private boolean ignoreLength;
 
+    // UTF-8 bytes, grows on demand. Actual length is dictated by bytesLength field.
```

Comment:
Would be nice to have some tests for StringSchemaKey

Reply:
Yup, agreed.

Reply:
Added some testing for dereferencing now, you think it's enough?